kitsune/journal/admin.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib import admin
+
+from kitsune.journal.models import Record
+
+
+class RecordAdmin(admin.ModelAdmin):
```

Comment:
Love having this app. Is the admin the only way to look at the data for now? Is that good enough?